src/braket/simulator/braket_simulator.py

```diff
@@ -59,6 +62,56 @@ def run(
             representing the results of the simulation.
         """
 
+    def run_multiple(
+        self,
+        payloads: Sequence[Union[OQ3Program, AHSProgram, JaqcdProgram]],
+        args: Optional[Sequence[Sequence[Any]]] = None,
+        kwargs: Optional[Sequence[Mapping[str, Any]]] = None,
+        max_parallel: Optional[int] = None,
+    ) -> list[Union[GateModelTaskResult, AnalogHamiltonianSimulationTaskResult]]:
+        """
+        Run the tasks specified by the given IR payloads.
+
+        Extra arguments will contain any additional information necessary to run the tasks,
+        such as number of shots.
+
+        Args:
+            payloads (Sequence[Union[OQ3Program, AHSProgram, JaqcdProgram]]): The IR representations
+                of the programs
+            args (Optional[Sequence[Sequence[Any]]]): The positional args to include with
+                each payload; the nth entry of this sequence corresponds to the nth payload.
+                If specified, the length of args must be equal to the length of payloads.
+                Default: None.
+            kwargs (Optional[Sequence[Mapping[str, Any]]]): The keyword args to include with
+                each payload; the nth entry of this sequence corresponds to the nth payload.
+                If specified, the length of kwargs must be equal to the length of payloads.
+                Default: None.
+            max_parallel (Optional[int]): The maximum number of payloads to run in parallel.
+                Default is the number of CPUs.
+
+        Returns:
+            list[Union[GateModelTaskResult, AnalogHamiltonianSimulationTaskResult]]: A list of
+            result objects, with the ith object being the result of the ith program.
+        """
+        max_parallel = max_parallel or cpu_count()
```

Comment:
With an optional parameter, most parallelization will use `-1` for cpu_count and default to 1. I think have the default be 1 (I could buy the argument of `min(2, cpu_count)`)

Reply:
From the [Python docs](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.pool.Pool), `multiprocessing.Pool` uses `None` for `cpu_count`. As well, a default of 1 would mean defaulting to single-threaded runs, which seems to defeat the purpose of `run_batch`.